testplan/web_ui/testing/src/Report/__tests__/reportUtils.test.js

```diff
@@ -70,7 +70,7 @@ describe('Report/reportUtils', () => {
     [
       [
         'testplan',
-        new Set([
```

Comment:
Reports can no longer use non-plain javascript types like `Set`. See [why](https://redux.js.org/style-guide/style-guide#use-plain-javascript-objects-for-state).